threat_intel/util/http.py

```diff
@@ -5,16 +5,55 @@
 # RateLimiter helps to only make a certain number of calls per second.
```

Comment:
It would be good to update the comment here as well about an additional class in the module.